src/pcapi/routes/serialization/reimbursement_csv_serialize.py

```diff
@@ -65,6 +66,7 @@ def __init__(self, payment_info: namedtuple = None):
             self.user_first_name = payment_info.user_firstName
             self.booking_token = payment_info.booking_token
             self.booking_used_date = payment_info.booking_dateUsed
+            self.booking_amount = payment_info.booking_amount * payment_info.booking_quantity
```

Comment:
```suggestion
            self.booking_total_amount = payment_info.booking_amount * payment_info.booking_quantity
```
Pour conserver la nomenclature utilisée dans la propriété du modèle `Booking`, et dissiper les doutes.